ui/chat/src/discussion.ts

```diff
@@ -84,11 +84,18 @@ function renderInput(ctrl: Ctrl): VNode | undefined {
 let mouchListener: EventListener;
 
 const setupHooks = (ctrl: Ctrl, chatEl: HTMLElement) => {
-  chatEl.addEventListener('keypress',
+  chatEl.addEventListener('keydown',
     (e: KeyboardEvent) => setTimeout(() => {
       const el = e.target as HTMLInputElement,
         txt = el.value,
         pub = ctrl.opts.public;
+      if(sessionStorage.getItem("txt") != null && sessionStorage.getItem("txt") != "") {
```

Comment:
I know little about this, but lichess appears to have a custom wrapper around `localStorage` and `sessionStorage` here: https://github.com/ornicar/lila/blob/master/ui/site/src/standalones/util.js#L76

I'm guessing you should use this